lib/services/fetch_warehouse.rb

```diff
@@ -0,0 +1,9 @@
+module Store
+  class FetchWarehouse
```

Comment:
wrong class name, change to sth better (example: https://github.com/gotar/internship_day_V)